README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
+e.g. [TEAMMATES](https://github.com/teammates/teammates), [SE-EDU projects](https://github.com/se-edu/main), 
+[PowerPointLabs](https://github.com/powerpointlabs/powerpointlabs).
 
-**Contents**
+Other projects are welcome to adopt/adapt these practices if they wish to.
 
-* [Guidelines for New Contributors](#guidelines-for-new-contributors)
-* [Coding Standards](#coding-standards)
-* [Naming Conventions](#naming-conventions)
-* [Git Usage Guidelines](#git-usage-guidelines)
-* [Contributing to this Repo](#contributing-to-this-repo)
+**For new contributors**:
 
-# Guidelines for New Contributors
+* [**Advice for OSS beginners**](docs/AdviceForOssBeginners.md) 
+* [**Asking a Question**](docs/QuestionsIssuesPrs.md#asking-a-question)
+* [**Posting an Issue**](docs/QuestionsIssuesPrs.md#posting-an-issue)
+* [**Submitting a PR**](docs/QuestionsIssuesPrs.md#submitting-a-pr) 
+* [**Advanced Contributor Guidelines**](docs/AdvancedContributorGuidelines.md) : Additional guidelines more 
```

Comment:
I personally prefer no whitespace before `:`, but it's personal so it does not need to be followed. The more important thing is, there are variations of `abc: def` and `abc : def` throughout this repo.